### DISCUSSION TITLE
Rdio scanner no link

### DISCUSSION THREAD

**Discussion by Dfowler267** [chuot/rdio-scanner#365] [General]
Hey I’m running the Rdio scanner app on iOS and just yesterday it seems like it just  shut down and won’t reconnect the scanner url is https://scanner.ecitez.com/ the access code I don't remeber if anyone could help me I would love the help.

**Comment by wmg819** [comment]
Just clicked on link and it ran fine for me. If you need to access the admin side use 
C:\Users\rdio\rdio-scanner>rdio-scanner -admin_password in a cmd prompt. Just make sure the path C:\Users\rdio\rdio-scanner> part points to the actual path your rdio-scanner is stored at which will allow you to reset the password.

**Comment by ldentel05** [comment]
Can you help me set up my app on iOS please?
